src/Features/Core/Portable/ExtractMethod/InsertionPoint.cs

```diff
@@ -1,45 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Linq;
-using Roslyn.Utilities;
-
-namespace Microsoft.CodeAnalysis.ExtractMethod;
-
-internal sealed class InsertionPoint
-{
-    private readonly SyntaxAnnotation _annotation;
-    private readonly Lazy<SyntaxNode?> _context;
```

Comment:
unneeded type that existed to just hang onto an object and an annotation.  like other places here and in previous prs, the annotation becomes static as there is only one of them ever.